lib/DDG/Spice/Tvmaze/Nextepisode.pm

```diff
@@ -0,0 +1,36 @@
+package DDG::Spice::Tvmaze::Nextepisode;
+
+use DDG::Spice;
+
+# the next episode will change the moment an episode has aired, so cache it for just one hour
+spice is_cached => 1;
+spice proxy_cache_valid => "200 301 302 404 1h";
```

Comment:
Ah, the 301 and 302 could be removed now that we're no longer using an endpoint that redirects. The 404 was intentional though: if no match was found for your query, the API will return a 404. Such a result should be cached just the same, I think.